arXiv.org.js

```diff
@@ -7,9 +7,19 @@
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
-	"translatorType": 4,
+	"translatorType": 8,
```

Comment:
Pretty sure this would disable web translation for arXiv.org — make sure you've tested that. I believe a combo web/search translator [should be `12`](https://www.zotero.org/support/dev/translators#metadata).